packages/web-twig/src/Resources/components/Alert/stories/AlertColors.twig

```diff
@@ -12,6 +12,7 @@
   <strong>Warning!</strong> Data update failed due to missing internet connection. See <a href="./" class="link-underlined">FAQ</a> for more info.
 </Alert>
 
-<Alert color="danger">
+{# @TODO: remove the `iconName` prop in https://jira.almacareer.tech/browse/DS-1177 #}
+<Alert color="danger" iconName="danger">
```

Comment:
I was not able to create the same magic as in the Alert web-react component. So as a fallback, I stayed with the `warning` icon. But in demos, we need to use the `danger` icon to have compatible snapshots between E-2-E tests. The Alert component in the web-twig cannot get the information about available icons directly. So that is why is it here as it is.